packages/create-catalyst/src/index.ts

```diff
@@ -38,11 +39,16 @@ const createCommand = program
   .description('Command to scaffold and connect a Catalyst storefront to your BigCommerce store')
   .option('--project-name <name>', 'Name of your Catalyst project')
   .option('--project-dir <dir>', 'Directory in which to create your project', process.cwd())
-  .option('--gh-ref <url>', 'Clone a specific ref from the bigcommerce/catalyst repository', 'main')
   .option('--store-hash <hash>', 'BigCommerce store hash')
   .option('--access-token <token>', 'BigCommerce access token')
   .option('--channel-id <id>', 'BigCommerce channel ID')
   .option('--customer-impersonation-token <token>', 'BigCommerce customer impersonation token')
+  .addOption(
+    new Option(
+      '--gh-ref <ref>',
+      'Clone a specific ref from the bigcommerce/catalyst repository',
+    ).default(getLatestCoreTag()),
```

Comment:
Won't this always fetch from gh even if `--gh-ref` is present? Thoughts on changing the default to only be the fn reference and only calling it when needed?

Reply:
Good catch!